src/pyedb/dotnet/edb_core/edb_data/simulation_configuration.py

```diff
@@ -1902,6 +1906,56 @@ def mesh_sizefactor(self, value):
             if value > 0.0:
                 self._do_lambda_refinement = False
 
+    @property
+    def adaptive_type(self):
+        """HFSS adaptive type.
+
+        Returns
+        -------
+        class: pyedb.dotnet.edb_core.edb_data.hfss_simulation_setup_data.AdaptiveType
+        """
+        return self._adaptive_type
+
+    @adaptive_type.setter
+    def adaptive_type(self, value):
+        if isinstance(value, int):
+            if value == 0:
+                self._adaptive_type = AdaptiveType.SingleFrequency
+            elif value == 1:
+                self._adaptive_type = AdaptiveType.MultiFrequency
+            elif value == 2:
+                self._adaptive_type = AdaptiveType.BroadBand
```

Comment:
```suggestion
        if isinstance(value, int) and value in range(3):
            self._adaptive_type = value
```
The values associated to AdaptiveType.whatever are already equal to the value passed as input

Reply:
@svandenb-dev Do we commit this changes and merge the PR or are there extra things to be done ?